src/modules/forms/plugins/RenderPlugins.js

```diff
@@ -86,6 +107,7 @@ class RenderPlugin extends Component {
                                         {...opts}
                                     />
                                 );
+                                {/*@TODO: FIX SOMETHING WOTH REQUEST TRANSFORM*/}
```

Comment:
typo `WOTH`

Reply:
Also prefer camel case comments `// FIXME unexpected error while transforming`

Reply:
Thank you!!! I just totally forgot to remove outdated comment 👍